bindings/pyroot/src/Pythonize.cxx

```diff
@@ -976,9 +976,7 @@ namespace {
       vi->vi_len = vi->vi_pos = 0;
       vi->vi_len = PySequence_Size( v );
 
-#ifndef R__WIN32 // prevent error LNK2001: unresolved external symbol __PyGC_generation0
-      _PyObject_GC_TRACK( vi );
```

Comment:
another possibility here is
```
-      _PyObject_GC_TRACK( vi );
+      PyObject_GC_Track( vi );
```

Reply:
Yes, but why not call PyObject_GC_Track on windows too? The #ifndef R__WIN32 was not always there. Before commit 9bb8349ee631929321a609032fc7c6f52891a637 the _PyObject_GC_TRACK macro was called on windows too, and the windows and linux code were the same. The missing symbol problem on windows should have been fixed by replacing _PyObject_GC_TRACK with PyObject_GC_Track instead of #ifdefing it out. If it had, the issue now seen with Python 3.7 on linux would already have been fixed. Removing the #ifndef in the PR was intentional, as stated in the PR description.

Reply:
Sorry I didn't see the `PyObject_GC_Track( vi );` being added below and thought this was just removing the entire block.